hack/get-machine-run-tests.sh

```diff
@@ -123,108 +150,146 @@ ROOT_VOLUME_SIZE=${ROOT_VOLUME_SIZE:-75}
 
 INSTNAME=${INSTNAME:-origin_$USER-$TESTNAME-$OS-1}
 
+# assumes origin version same as origin-aggregated-logging version same
+# as openshift-ansible version
+case $GIT_BASE_BRANCH in
+    master) oshift_ver=4.0 # master currently corresponds to 4.0
+            OPENSHIFT_IMAGE_TAG=${OPENSHIFT_IMAGE_TAG:-v$oshift_ver} ;;
+    release-*) oshift_ver=$( echo $GIT_BASE_BRANCH | sed 's/.*-\([0-9][0-9]*[.][0-9][0-9]*\)/\1/' )
+               OPENSHIFT_IMAGE_TAG=${OPENSHIFT_IMAGE_TAG:-v$oshift_ver} ;;
+    *) oshift_ver=master
+       OPENSHIFT_IMAGE_TAG=${OPENSHIFT_IMAGE_TAG:-latest} ;;
+esac
 
 pushd $GIT_REPO_BASE_DIR/origin-aggregated-logging
 
-# https://github.com/openshift/origin-ci-tool#installation
-NO_SKIP=${NO_SKIP:-0}
-if [ ! -d .venv ] ; then
-    virtualenv .venv --system-site-packages
-    NO_SKIP=1
-fi
-PS1=unused
-source .venv/bin/activate
-if [ "${NO_SKIP:-0}" = 1 ] ; then
-    if pip show origin-ci-tool > /dev/null ; then
-#        pip install --upgrade git+file://$HOME/origin-ci-tool --process-dependency-links
-        pip install --upgrade git+https://github.com/openshift/origin-ci-tool.git --process-dependency-links
-    else
-#        pip install git+file://$HOME/origin-ci-tool --process-dependency-links
-        pip install git+https://github.com/openshift/origin-ci-tool.git --process-dependency-links
+if [ $PROVIDER = aws ] ; then
+    export REMOTE_GOPATH=${REMOTE_GOPATH:-/data}
```

Comment:
is there a reason we `export` the same value twice? (also line 122)

Reply:
fixed